Cargo.toml

```diff
@@ -21,6 +21,8 @@ unwrap = "~1.2.1"
 [dev-dependencies]
 hamcrest2 = "~0.2.3"
 mio-extras = "~2.0.5"
+net-literals = "~0.1.2"
```

Comment:
i can't seem to find in this PR what macro (or if anything else) is used from here. I'm very heavily biased towards _not_ using this so do remove it unless you have a very strong reason to use it.

Reply:
It's the `addr!` macro: https://github.com/maidsafe/socket-collection/pull/18/commits/2daa0932c4b75e554915b9274394f059c563d275#diff-54b812c3690afc1f582662413b8f058eR21

Reply:
I removed it, although I do love the idea: `addr!` and other related macros check IP addresses at compile time - saves one `unwrap!` statement :)